e2e/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "firebase-smoke-test",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "setup": "node test-setup.js",
+    "test": "yarn setup && karma start --compat --exp",
```

Comment:
no test command for v8?

Reply:
I don't know if it's worth maintaining v8 e2e tests for the occasional GCIP update?